tools/parse_map_file.py

```diff
@@ -6,18 +6,40 @@
 from enum import Enum
 
 class LinkerSection(Enum):
+    """Enum to identify the section of the map file currently being parsed."""
     UNKNOWN = 1
     TEXT = 2
     DATA = 3
     BSS = 4
 
 linker_section_ids = {
+    """Dictionary to map map-file section name to enum identifier."""
     ".text": LinkerSection.TEXT,
     ".data": LinkerSection.DATA,
     ".bss": LinkerSection.BSS
 }
 
 def read_gnu_map_file(file_name):
```

Comment:
This is doing a little more than "reading". looks like it's doing some parsing or grabbing/importing.... I'm not sure what word to use, but I think of "read" as a first step to manipulating the data in some way, which this seems to be doing.

Reply:
Perhaps `parse_gnu_map_file`?